CHANGELOG.md

```diff
@@ -4,6 +4,109 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.28.0] - 2023-08-24
+
+## 🚀 Features
+
+### Expose the stats_reports_key hash to plugins. ([Issue #2728](https://github.com/apollographql/router/issues/2728))
+
+This changeset exposes a new key in the Context, `apollo_operation_id`, which identifies operation you can find in studio:
+
+```
+https://studio.apollographql.com/graph/<your_graph_variant>/variant/<your_graph_variant>/operations?query=<apollo_operation_id>
+```
+
+The `apollo_operation_id` context key is exposed during:
+
+- Execution service request
+- Subgraph service request
+- Subgraph service response
+- Execution service response
+- Supergraph service response
+- Router service response
+
+By [@o0Ignition0o](https://github.com/o0Ignition0o) in https://github.com/apollographql/router/pull/3586
+
+### Add new (unstable) metrics ([PR #3609](https://github.com/apollographql/router/pull/3609))
+
+Many of our existing metrics are poorly and inconsistently named. In addition, they follow Prometheus style rather than Otel style.
+
+This changeset adds some new metrics that will give us a good foundation to build upon.
+New metrics are namespaced `apollo.router.operations.*`.
+
+These metrics should be treated as unstable and may change in the future.
+
+By [@BrynCooke](https://github.com/BrynCooke) in https://github.com/apollographql/router/pull/3609
+
+## 🐛 Fixes
+
+### Flush metrics when Router reloads or shuts down ([Issue #3140](https://github.com/apollographql/router/issues/3140))
+
+When the Router either reloads or shuts down it now flushes metrics.
+Push metrics exporters such as OTLP would have previously missed some metrics, in particular thouse related to reload events.  
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/3143
+
+## 🛠 Maintenance
+
+### Enable checking for kubernetes 1.28.0 in kubeconform ([Issue #3587](https://github.com/apollographql/router/issues/3587))
+
+Kubeconform now supports kubernetes `1.28.0` and we have re-enabled manifest verification in CI.
+
+This reverts #3584.
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/3638
+
```

Comment:
While I could propose edits, I don't think it's worth the time.  I propose that we remove this entry for a few reasons:

- We actually landed the reversion between releases, so there was never a change log that said we had it in the first place.
- We don't have a history of actually announcing this compatibility.  We should do that! But we should do it consistently probably.
- Even more specifically:
  - "we can re-enable checking" — I suspect a user might be led to ask, "_re-enabling checking of what_"? Or even "What’s `kubeconform` and what does it mean to me in this release?"
     Adding context could fix this, e.g., “Previously, we temporarily disabled ___ because ___.  This change puts that back in place.", but again historically we didn't so opportunity for next time, maybe.
- "Enable checking for Kubernetes 1.28.0" (in the title) doesn’t seem to explain _what_ is happening now either.  _What are we are doing?_   My own understanding is that we’re _validating that our Helm charts work with versions of Kubernetes_?  (Perhaps let’s say that?).  As another title, maybe `Validate Helm charts against Kubernetes v1.28.0`? 

Anyhow, because of my first two bullets, I think we just remove this.

```suggestion
```